ui/lib/kv/addon/components/page/secret/paths.hbs

```diff
@@ -0,0 +1,65 @@
+<KvPageHeader @breadcrumbs={{@breadcrumbs}} @pageTitle={{@path}}>
+  <:tabLinks>
+    <LinkTo @route="secret.details" data-test-secrets-tab="Secret">Secret</LinkTo>
+    <LinkTo @route="secret.metadata.index" data-test-secrets-tab="Metadata">Metadata</LinkTo>
+    <LinkTo @route="secret.paths" data-test-secrets-tab="Paths">Paths</LinkTo>
+    {{#if @secret.canReadMetadata}}
+      <LinkTo @route="secret.metadata.versions" data-test-secrets-tab="Version History">Version History</LinkTo>
+    {{/if}}
+  </:tabLinks>
+</KvPageHeader>
+
+<h2 class="title is-5 has-top-margin-l">
+  Paths
+</h2>
+
+<div class="box is-fullwidth is-sideless is-paddingless is-marginless">
+  {{#each this.paths as |path|}}
+    <InfoTableRow @label={{path.label}} @labelWidth="is-one-third" @helperText={{path.text}}>
+      {{! replace with Hds::Copy::Snippet }}
+      <CopyButton
+        class="button is-compact is-transparent level-right"
+        @clipboardText={{path.snippet}}
+        @buttonType="button"
+        @success={{action (set-flash-message (concat path.label " copied!"))}}
```

Comment:
I believe the Octane pattern is to avoid the action modifier and use the fn instead. [See here](https://ember-learn.github.io/ember-octane-vs-classic-cheat-sheet/#actions) on the cheat sheet.

Actually, the only example of this in the code on this component is from you :)
```
 @success={{fn (set-flash-message "JSON Copied!")}}
```

Reply:
Ah yeah - this was an old pattern that I just copied - I can update!